fetch.bs

```diff
@@ -3123,58 +3143,147 @@ response <a for=/>header</a> can be used to require checking a <a for=/>request<
 Cross-Origin-Resource-Policy     = %s"same-origin" / %s"same-site" / %s"cross-origin" ; case-sensitive
 </code></pre>
 
-<p>To perform a <dfn>cross-origin resource policy check</dfn>, given a <var>request</var> and
-<var>response</var>, run these steps:</p>
+<p>To perform a <dfn>cross-origin resource policy internal check</dfn>, given an
```

Comment:
I don't think this algorithm works now we also use it for responses from a service worker as it primarily uses the request as authority, whereas the request has almost zero authority in such situations.

This is similar to the problem with the Cache API check and needs a similar solution.